### DISCUSSION TITLE
strongswan lost some config files when start service

### DISCUSSION THREAD

**Discussion by sungguoshuai** [strongswan/strongswan#257] [General]
Hello,
I have installed strongswan 5.7.2 on Linux. When I start the strongswan service using systemd, I notice that the service log shows charon can not find some config files. Are these missing config files a problem or just a warning? I am not familiar with strongswan, so if anyone can tell me about this, I will be happy.
`charon[3155]: 00[CFG] opening triplet file /etc/strongswan/ipsec.d/triplets.dat: No such file or firectory`
`charon[3155]: 00[TNC] opening configuration file '/etc/tnc_config' failed: No sunch file or directory`

**Comment by Thermi** [comment]
Hi,

These warnings are caused by plugins that you load but, at the very least it seems like it, not need. E.g. those are one of the TNC and one EAP-AKA or -SIM related plugin. You're very likely not using them, so you can ignore them.
